pkgs/development/python-modules/tls-parser/default.nix

```
@@ -0,0 +1,20 @@
+{ stdenv, lib, fetchFromGitHub, buildPythonPackage }:
+
+buildPythonPackage rec {
+  name = "tls-parser";
+  version = "1.2.1";
```

usually white space between version and src
```suggestion
  version = "1.2.1";
  
```